lm_eval/models/huggingface.py

```diff
@@ -1104,7 +1130,9 @@ def _lookup_one_token_cont(req: Tuple[Tuple[str, str], List[int], List[int]]):
                 }
 
             multi_logits = F.log_softmax(
-                self._model_call(batched_inps, **call_kwargs), dim=-1
+                self._model_call(batched_inps, **call_kwargs),
+                dim=-1,
+                # dtype=torch.float16,
```

Comment:
I suggest removing this commented out code

Reply:
I would like to ask @haileyschoelkopf about this line. We added to avoid OOM issues but it changes the results slightly and makes the tests break. This is a valuable add imo as I don't think we need `float32` for this but tell me what you think !

Reply:
dtype for HFLM can be passed though `--model_args`, so if one specifically set something like `pretrained=EleutherAI/pythia-160m,dtype=float32` or bfloat16, it should be followed, not hidden change for calculations of batch size